Sources/SwiftProtobuf/TextScanner.swift

```diff
@@ -15,55 +15,227 @@
 import Foundation
 import Swift
 
+private func fromHexDigit(_ c: UInt8) -> UInt8? {
+  if c >= 48 && c <= 57 {
```

Comment:
Thanks for the nudge:  I've been chafing about how illegible all these numbers are.

I just went through and added symbolic constants for all of them.